### ISSUE TITLE
[HPC_AI][AI for Science Climate] Text pre-processing notebook throws pandas error

### ISSUE THREAD

**Issue by pzharrington** [openhackathons-org/gpubootcamp#111]
In file `Pre-Processing_Text_data.ipynb`, there is a `pandas` call that is out of date for `pandas>=1.4`, but the Dockerfile does not specify a specific `pandas` version during installation. The line in question is:
```
pd.set_option("max_columns", None)
```

Should be updated to 
```
pd.set_option("display.max_columns", None)
```

Otherwise, `pandas<1.4` should be specified in the Dockerfile. See https://github.com/pandas-dev/pandas/issues/45619

**Comment by bharatk-parallel** [comment]
Valid observation. Will be updated

**Comment by aswinkumar1999** [comment]
Thank you @pzharrington for bringing it up.

I've created a pull request #112 wherein I had set the pandas version explicitly to 1.3.5 and had built and tested the container for reliability. 

Requesting @bharatk-parallel to review and merge the request.

**Comment by aswinkumar1999** [comment]
Fixed in #112 

Closing issue.
